redact/gatsby-config.js

```diff
@@ -23,11 +23,15 @@ module.exports = {
         sidebarCategories: {
           null: [
             "index",
-            "reference",
           ],
-          Redactors: [
+          "Default Redactors": [
```

Comment:
Let's keep this as `Redactors`. We will be extending this section to match Collectors and Analyzers where the list will be the built-in functions that can be extended.